data/projects.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
Placeholder data should be replaced with actual content.

Update the properties to be the final version for what you're going to have on your portfolio preview and project detail sections.